internal/types/constants/trailers.go

```diff
@@ -0,0 +1,28 @@
+package constants
+
+const (
```

Comment:
I'd suggest sorting this list of constants alphabetically, since that's how they'll show up in commit messages.